app/views/account_reset/confirm_request/show.html.erb

```diff
@@ -2,10 +2,16 @@
 
 <%= render PageHeadingComponent.new.with_content(t('headings.verify_email')) %>
 
-<p><%= t('account_reset.confirm_request.instructions_start') %>
-  <strong><%= email %></strong>
-  <%= t('account_reset.confirm_request.instructions_end') %>
-</p>
+<% if IdentityConfig.store.updated_account_reset_content %>
```

Comment:
This might be a bit nit-pick-y, but can we make `IdentityConfig` values a variable in the controller that describes what the config value does?